downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -27,6 +27,29 @@ Update your current Ballerina installation directly to 2201.7.0 by using the [Ba
 If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
 
 ## Backward-incompatible changes
+- Fixed a bug that sets broad type for constants instead of singleton type.
+
+  ```ballerina
+  const int CONST = 1 + 7;
+  CONST num = 2;
+  ```
+  Now, the code produces an error: `incompatible types: expected '8', found 'int'`. Similarly, the following assignment in the function scope also results in an error: `incompatible types: expected 'SingletonType', found 'int'`.
+
+  ```ballerina
+  const int CONST = 3 + 5;
+  type SingletonType CONST;
+  
+  public function main() {
+    SingletonType num = 3;
+  }
+  ```
```

Comment:
Do we really need this second example? The first one conveys the change, right?

Reply:
Yes. Removed